docs/configuration.md

```diff
@@ -1094,43 +1094,117 @@ var config = {
 >
 > ```js
 > {
->   adBreaks: Array<PKAdBreakObject>,
->   playAdsAfterTime?: number
+>   playAdsAfterTime?: number,
+>   adBreaks: Array<PKAdBreakObject>
 > }
 > ```
 >
 > ##### Default: `-`
 >
 > ##### Description: Defines an advertising scheme (optional).
 >
+> ##
+>
+> > ### config.advertising.playAdsAfterTime
+> >
+> > ##### Type: `number`
+> >
+> > ##### Default: `config.playback.startTime`
+> >
+> > ##### Description: Only play ad breaks scheduled after this time (in seconds). This setting is strictly after - e.g. setting playAdsAfterTime to 15 will cause the player to ignore an ad break scheduled to play at 15s.
+> >
 > > ##
 > >
 > > ### config.advertising.adBreaks
 > >
 > > ##### Type: `Array<PKAdBreakObject>`
 > >
+> > ##### Description: The ad breaks scheme
+> >
+> > ##
+> >
 > > > ##### Type `PKAdBreakObject`
 > > >
 > > > ```js
 > > > {
 > > >   position: number,
+> > >   percentage?: number, // optional
+> > >   every?: number, // optional
 > > >   ads: Array<PKAdObject>
 > > > }
 > > > ```
 > > >
-> > > > ##### Type `PKAdObject`
+> > > ##
+> > >
+> > > > ##### `PKAdBreakObject.position`
 > > > >
-> > > > ```js
-> > > > {
-> > > >   url?: Array<string>,
-> > > >   response?: Array<string>,
-> > > >   bumper?: boolean
-> > > > }
-> > > > ```
-> >
-> > ##### Default: `[]`
+> > > > ##### Type: `number`
+> > > >
+> > > > ##### Description: The ad break position in seconds.
+> > > >
+> > > > ##
+> > > >
+> > > > ##### `PKAdBreakObject.percentage`
+> > > >
+> > > > ##### Type: `number`
+> > > >
+> > > > ##### Description: The ad break position in percentage of the media length instead of by seconds (optional).
+> > > >
```

Comment:
I don't understand. If given percentage then the position is not relevant? I think this needs to be clarified

Reply:
giving percentage is instead of giving a position